m/murmur/murmur_1.0.0_ubi_8.4.sh

```diff
@@ -0,0 +1,61 @@
+# ----------------------------------------------------------------------------
+#
+# Package               : murmur
+# Version               : murmur-1.0.0
+# Source repo           : https://github.com/sangupta/murmur/
+# Tested on             : UBI 8.4
+# Language              : Java
+# Travis-Check          : False
+# Script License        : Apache License, Version 2 or later
+# Maintainer            : Vikas <kumar.vikas@in.ibm.com>
+#
+# Disclaimer            : This script has been tested in root mode on given
+# ==========              platform using the mentioned version of the package.
+#                         It may not work as expected with newer versions of the
+#                         package and/or distribution. In such case, please
+#                         contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+set -e
+
+if [ -z "$1" ]; then
+  export PACKAGE_VERSION=murmur-1.0.0
+else
+  export PACKAGE_VERSION=$1
+fi
+if [ -d "murmur" ] ; then
+  rm -rf murmur
+fi
+
+# Dependency installation
+dnf install -y git java-1.8.0-openjdk-devel maven
```

Comment:
We don't need to install `java-1.8.0-openjdk-devel` explicitly, if we are installing maven.
It will get installed as a dependency for Maven.